storage/kv.go

```diff
@@ -35,6 +35,7 @@ type KV interface {
 	// If `end` is not nil, it gets the keys in range [key, range_end).
 	// Limit limits the number of keys returned.
 	// If the required rev is compacted, ErrCompacted will be returned.
+	// The returned rev is the current revision of the KV when the operation is executed.
```

Comment:
Yeah just moved it before error docs, and added it to other methods (Put, DeleteRange).